lib/sidekiq/web/csrf_protection.rb

```diff
@@ -90,13 +90,11 @@ def valid_token?(env, giventoken)
         end
 
         sess = session(env)
-
-        # Checks that Rack::Session::Cookie did not return empty session
-        # object in case the digest verification failed
-        return false if sess.empty?
-
         localtoken = sess[:csrf]
 
+        # Checks that Rack::Session::Cookie actualy contains the csrf toekn
+        return false if localtoken.to_s.strip.empty?
```

Comment:
Why not just `return false if localtoken.nil?`

Reply:
true, i guess protecting against an empty string isn't necessary here